docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -270,3 +285,37 @@ For information about exceptions and how to use them, refer to
 === Visually analyze process relationships
 
 For process events that are detected by {elastic-endpoint}, you can open a visual mapping to view a hierarchal timeline of when these events occurred. For more information, refer to <<visual-event-analyzer>>.
+
+[float]
+[[alerts-run-osquery]]
+=== Run Osquery
+Run live queries against the alert's host to learn more about your infrastructure and operating systems.
+
+IMPORTANT: The host associated with the alert must have the Osquery integration enabled. In addition, to use Osquery, your role requires specific privileges. Refer to {kibana-ref}/osquery.html[Osquery] for more information.
```

Comment:
Can we specify that it's the **Osquery Manager** integration in this sentence, just to make sure it's clear? (there are 2 osquery integrations)

> The host associated with the alert must have the Osquery Manager integration enabled.